ref/test/test_output/c_sw_12x24.test

```diff
@@ -3,82 +3,82 @@ TEST          Input State
 TEST ===================================================================================================================
 TEST        Variable                 Min                 Max               First                Last                 RMS
 TEST -------------------------------------------------------------------------------------------------------------------
-TEST           rarea  -0.10000000000E-07   0.68896173601E-09  -0.10000000000E-07   0.56546800912E-09   0.14365610413E-08
-TEST         rarea_c   0.56234680356E-09   0.69052250213E-09   0.67316411530E-09   0.56234680356E-09   0.63558157249E-09
```

Comment:
Ouch, this PR increases the accuracy from 11 digits to 12 digits.  When I added interpolation to SENA-c_sw (not a PR yet) I was having troubling getting 11 digits of accuracy.  12 digits will be troublesome.

Reply:
The increase was unintentional.  @robgonzalezpita, could you please change the `5ES20.11` to `5ES20.10` and regenerate new baselines?  @LyndStringer is correct.

Reply:
@christopherwharrop-noaa, you're right, the increased accuracy was indeed unintentional. I created this PR knowing there would be some necessary adjustments. @LyndStringer,  I'll make the changes here, as well as in the outstanding  PR for YPPM regarding this issue.